README.md

```diff
@@ -5,21 +5,58 @@
 
 This is a simple library that generates SVG badges without Cairo.
 
+
 ## Install
 
 `npm install badge-up`
 
-## Usage
 
-### Generating an SVG
+## V1 Usage
 
 ```js
-
 var badge = require('badge-up');
-
 badge('batman', 'component', badge.colors.green, function (error, svg) {
     // some callback
 });
 ```
 
 Produces: ![example](https://cdn.rawgit.com/yahoo/badge-up/master/test/testData/good.svg)
+
+The color argument can be a CSS color, or one of the specially named colors
+found in `badge.colors`.
+
+
+## V2 Usage
+
+With this API you can create arbitrary badges with an arbitrary number of sections.
+
+Each section is an array. The first element is the text to show, and the rest
+of the (optional) elements are attributes to apply to the text. If there are no
+attributes then a single string (instead of an array of one element) can be
+passed instead.
+
+The following attributes are supported:
+
+* `{hex}{hex}{hex}{hex}{hex}{hex}` six hexadecimal characters or `{css-color-name}` a named CSS color,
+    either of which is used to color the section
+
+The default color for the first section is `696969` and `d3d3d3` for subsequent sections.
+
+Any section can have mulitple lines by putting newlines in its text.
+
+```js
+var badge = require('badge-up');
+var sections = [
+        'foo/far;fun',
+        [ 'bar\nbaz', 'orange'],
+        [ 'mork "mindy"', 'olive'],
+        [ '<∀>', 'moccasin']
+    ];
+badge.v2(sections, function (error, svg) {
+    // some callback
+});
+```
+
+Produces: ![example](https://cdn.rawgit.com/yahoo/badge-up/master/test/testData/v2-example.svg)
```

Comment:
Yeah it won't work until this PR is merged, since it's a pointing to a file added by this PR. Here's the pre-merge version:
https://github.com/drewfish/badge-up/blob/921d332544392b95b5127f067107fe93c609bc41/test/testData/v2-example.svg